backend/src/achievements/repository/achievements.repository.ts

```diff
@@ -0,0 +1,12 @@
+import { Achievement } from '../entities/achievements.entity';
+import { Injectable } from '@nestjs/common';
+import { EntityRepository, Repository } from 'typeorm';
+import { ResponseAchievementDto } from '../dto/response-achievement.dto';
+
+@Injectable()
+@EntityRepository(Achievement)
+export class AchievementRepository extends Repository<Achievement> {
+  async findAll(): Promise<ResponseAchievementDto[]> {
+    return await this.find();
+  }
```

Comment:
Did you talk with @fyusuf-a about paginated responses? It might be useful to implement that here

Reply:
We did, and for now the conclusion is -if  I recall correctly- that the Achievement Log (containing an achievement) would be the one we actually send to the client,and thus the one that need pagination.

Reply:
Okay, perfect!